digitransit-component/packages/digitransit-component-suggestion-item/src/index.js

```diff
@@ -90,7 +90,7 @@ function getIconProperties(
     iconColor = color;
   }
   const layerIcon = new Map([
-    ['bikeRentalStation', 'citybike'],
+    ['vehicleRentalStation', 'citybike'],
```

Comment:
Same as above, lets keep calling this bikeRentalStation because the icon has a bike.